src/benchmarks/gc/GC.Infrastructure/GC.Analysis.API/DynamicEvents/GCDynamicEvents.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace GC.Analysis.API.DynamicEvents
```

Comment:
As part of our notebook-samples-todo, we should include the workflow for adding a new dynamic event.  I think it would be something patterned after this code and calling the `Add` method, but seeing it would help evalulate it.  (Andrew has talked about how it's a lot of work to use dynamic events.  This would make it explicit.  This PR contains some of the infra (e.g., the size of a uint) which isn't needed for new events.)

Reply:
To then evaluate this fairly, we'd want the "best" version possible of the initialization code.  This is subjective of course...
- `KeyValuePair.Create("version", typeof(ushort)` seems better
- C# initialization syntax for the list?
- consider a "data" format for the schema rather than C# code?
- consider having the build of the GC itself (which defines how to write the dynamic event) write a schema or maybe embed it in the etl file so that the info doesn't need to be manually written twice

Reply:
Addressed the comments that are in scope. Resolving as the discussion here is out of scope - but this should be captured in an issue.